server/src/main/java/org/opensearch/action/admin/cluster/decommission/awareness/put/DecommissionRequest.java

```diff
@@ -74,6 +103,12 @@ public ActionRequestValidationException validate() {
         if (decommissionAttribute.attributeValue() == null || Strings.isEmpty(decommissionAttribute.attributeValue())) {
             validationException = addValidationError("attribute value is missing", validationException);
         }
+        if (drainingTimeout.getSeconds() < 0 || drainingTimeout.getSeconds() > MAX_NODE_DRAINING_TIMEOUT.getSeconds()) {
```

Comment:
get millis?

Reply:
Using getSeconds only. Don't see any specific benefit and will make it confusing since we are using seconds everywhere in this flow.